From bcd67d02631187a724cb06ce762641a281d5168a Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Mon, 19 Oct 2015 17:34:54 +0800
Subject: [PATCH 0589/5242] MLK-11739-2 ARM: dts: imx6qdl: fec interrupt
 should be in GPC domain

commit  02241ea2bffac2d5d8edadcdd3aaefd5b51a32fb from
https://source.codeaurora.org/external/imx/linux-imx.git

GPC irq domain is child domain of GIC, all of platform
irqs should be inside GPC domain.

Signed-off-by: Anson Huang <b20788@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6dl-sabresd-enetirq.dts |    2 +-
 arch/arm/boot/dts/imx6q-sabresd-enetirq.dts  |    2 +-
 arch/arm/boot/dts/imx6qdl-sabreauto.dtsi     |    2 +-
 arch/arm/boot/dts/imx6qdl.dtsi               |    4 ++--
 4 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/imx6dl-sabresd-enetirq.dts b/arch/arm/boot/dts/imx6dl-sabresd-enetirq.dts
index b23d0eb..ff41441 100644
--- a/arch/arm/boot/dts/imx6dl-sabresd-enetirq.dts
+++ b/arch/arm/boot/dts/imx6dl-sabresd-enetirq.dts
@@ -10,7 +10,7 @@
 
 &fec {
 	pinctrl-0 = <&pinctrl_enet &pinctrl_enet_irq>;
-	interrupts-extended = <&gpio1 6 0x04>, <&intc 0 119 0x04>;
+	interrupts-extended = <&gpio1 6 0x04>, <&gpc 0 119 0x04>;
 };
 
 &i2c3 {
diff --git a/arch/arm/boot/dts/imx6q-sabresd-enetirq.dts b/arch/arm/boot/dts/imx6q-sabresd-enetirq.dts
index 14b1cfa3..69da404 100644
--- a/arch/arm/boot/dts/imx6q-sabresd-enetirq.dts
+++ b/arch/arm/boot/dts/imx6q-sabresd-enetirq.dts
@@ -10,7 +10,7 @@
 
 &fec {
 	pinctrl-0 = <&pinctrl_enet &pinctrl_enet_irq>;
-	interrupts-extended = <&gpio1 6 0x04>, <&intc 0 119 0x04>;
+	interrupts-extended = <&gpio1 6 0x04>, <&gpc 0 119 0x04>;
 };
 
 &i2c3 {
diff --git a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
index 11b9130..69be2bd 100644
--- a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
@@ -408,7 +408,7 @@
 	pinctrl-0 = <&pinctrl_enet>;
 	phy-mode = "rgmii";
 	interrupts-extended = <&gpio1 6 IRQ_TYPE_LEVEL_HIGH>,
-			      <&intc 0 119 IRQ_TYPE_LEVEL_HIGH>;
+			      <&gpc 0 119 IRQ_TYPE_LEVEL_HIGH>;
 	fsl,err006687-workaround-present;
 	status = "okay";
 };
diff --git a/arch/arm/boot/dts/imx6qdl.dtsi b/arch/arm/boot/dts/imx6qdl.dtsi
index 03c88b86..01e5d0e 100644
--- a/arch/arm/boot/dts/imx6qdl.dtsi
+++ b/arch/arm/boot/dts/imx6qdl.dtsi
@@ -1098,8 +1098,8 @@
 				reg = <0x02188000 0x4000>;
 				interrupt-names = "int0", "pps";
 				interrupts-extended =
-					<&intc 0 118 IRQ_TYPE_LEVEL_HIGH>,
-					<&intc 0 119 IRQ_TYPE_LEVEL_HIGH>;
+					<&gpc 0 118 IRQ_TYPE_LEVEL_HIGH>,
+					<&gpc 0 119 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&clks IMX6QDL_CLK_ENET>,
 					 <&clks IMX6QDL_CLK_ENET>,
 					 <&clks IMX6QDL_CLK_ENET_REF>;
-- 
1.7.9.5

